content/fr/docs/concepts/workloads/pods/pod-qos.md

```diff
@@ -0,0 +1,68 @@
+
+### Burstable
+
+Les pods qui sont « Burstables » ont des garanties de ressources de limite inférieure basées sur la demande, maisn’exigent pas de limite spécifique. Si aucune limite n’est spécifiée, elle est définie par défaut surlimite équivalente à la capacité du nœud, ce qui permet aux pods d’augmenter de manière flexibleleurs ressources si des ressources sont disponibles. En cas d’expulsion du Pod en raison de NodePression sur les ressources, ces Pods ne sont expulsés qu’après que tous les Pods 'BestEffort' ont été expulsés.Étant donné qu’un pod « Burstable » peut inclure un conteneur qui n’a pas de limites de ressources ou de demandes, un podc’est-à-dire que 'Burstable' peut essayer d’utiliser n’importe quelle quantité de ressources de nœud.
+
+#### Critères
+
+Un Pod reçoit une classe QoS de « Burstable » si :
+
+- Le Pod ne répond pas aux critères de la classe QoS 'Garantie'.
+- Au moins un conteneur dans le Pod a une demande ou une limite de mémoire ou de CPU.
+
+### Meilleur effort
+
+Les pods de la classe QoS « BestEfforts » peuvent utiliser des ressources de nœud qui ne sont pas spécifiquement affectéesaux Pods d’autres classes QoS. Par exemple, si vous avez un nœud avec 16 cœurs de processeur disponibles pour lekubelet, et vous attribuez 4 cœurs CPU à un Pod 'Guaranteed', puis un Pod dans le 'BestEffort'La classe QoS peut essayer d’utiliser n’importe quelle quantité des 12 cœurs de processeur restants.Le kubelet préfère expulser les pods 'BestEffort' si le nœud est soumis à une pression sur les ressources.
+
+#### Critères
+
+A Pod has a QoS class of `BestEffort` if it doesn't meet the criteria for either `Guaranteed`
+or `Burstable`. In other words, a Pod is `BestEffort` only if none of the Containers in the Pod have a
+memory limit or a memory request, and none of the Containers in the Pod have a
+CPU limit or a CPU request.
+Containers in a Pod can request other resources (not CPU or memory) and still be classified as
+`BestEffort`.
+
+## Certains comportements sont indépendants de la classe QoS {#class-independent-behavior}
+
+Certains comportements sont indépendants de la classe QoS attribuée par Kubernetes. Par exemple:
+
+- Tout conteneur dépassant une limite de ressources sera tué et redémarré par le kubelet sansaffectant d’autres conteneurs dans ce pod.
+- Si un conteneur dépasse sa demande de ressources et que le nœud sur lequel il s’exécute fait facepression des ressources, le Pod dans lequel il se trouve devient un candidat pour [expulsion](/docs/concepts/scheduling-eviction/node-pressure-eviction/).Si cela se produit, tous les conteneurs du pod seront résiliés. Kubernetes peut créer unPod de remplacement, généralement sur un nœud différent.
+- La demande de ressource d’un Pod est égale à la somme des demandes de ressources deses conteneurs de composants et la limite de ressources d’un pod est égale à la somme deles limites de ressources de ses conteneurs composants.
```

Comment:
Incomprehensible, reformulate.

Reply:
I checked with the English version of the same and translating the above text in English is exactly similar to the English version. The translation here seems correct to me but do correct me.

English Version: https://github.com/kubernetes/website/blob/main/content/en/docs/concepts/workloads/pods/pod-qos.md

Reply:
I look this tonight